gammapy/makers/background/tests/test_reflected.py

```diff
@@ -246,3 +247,50 @@ def test_reflected_bkg_maker_no_off_background(reflected_bkg_maker, observations
 
     assert_allclose(datasets[0].counts_off.data, 0)
     assert_allclose(datasets[0].acceptance_off, 0)
+
+
+def test_wobble_regions_finder():
+    source_angle = 35 * u.deg
+
+    on_region = CircleSkyRegion(
+        center=SkyCoord.from_name("Crab"),
+        radius=0.15 * u.deg,
+    )
+    pointing = on_region.center.directional_offset_by(
+        separation=0.4 * u.deg,
+        position_angle=180 * u.deg + source_angle,
+    )
+
+    assert u.isclose(pointing.position_angle(on_region.center).to(u.deg), source_angle, rtol=0.05)
+    n_off_regions = 3
```

Comment:
What happens if e.g. `n_off_regions = 15` ? 
I suppose one wants non-overlapping regions, so that the `WobbleRegionsFinder` should fail or return no regions.

This should be tested.

Reply:
Do you think it should only be a warning or an error to have overlapping regions? Having them overlap is not wrong per se but it violates the poisson assumption made later when fitting since the counts in the single regions are not independent from each other anymore.

So raising an error would probably be the safest option.

Reply:
Ok, looking at the current code, it logs a warning and sets the counts to None, so maybe we return no (valid) regions found from the finder.